roles/sensu/server/tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: configure sensu
+  template:
+   src: 'templates/{{ item }}.json.j2'
+   dest: '{{sensu_config_path}}/{{ item }}.json'
+   owner: '{{ sensu_owner }}'
+   group: '{{ sensu_group }}'
+  with_items:
+      - api
+      - redis
+      - sensu_oschecks
+  notify:
+      - restart sensu
+
+- name: fix sensu_oschecks
```

Comment:
Thanks for the response on the previous patchset.  It would be great if you could leave a comment on this task that explains what it's doing, since otherwise it may mystify other people.